lib/ruby/truffle/jruby-truffle-tool/lib/truffle_tool.rb

```diff
@@ -518,6 +518,34 @@ def parse_gemfile_line(line)
     BUNDLER_EVAL_ENV.send :eval, line
   end
 
+  def gemfile_use_path!(gems_path)
+    gemfile = Bundler.default_gemfile.to_s
+
+    new_lines = File.read(gemfile).lines.map do |line|
+      if line =~ /^( +)gem.*git:/
+        space               = $1
+        gem_name, options   = parse_gemfile_line(line)
+        repo_name           = options[:git].split('/').last
+        repo_match          = "#{gems_path}/bundler/gems/#{repo_name}-*"
+        repo_path           = Dir[repo_match].sort.first
+        options_without_git = options.merge(path: repo_path).tap do |h|
+          h.delete(:git)
+          h.delete(:branch)
+        end
+
+        format(<<-RUBY.gsub(/^ +/, space), line.strip, gem_name, options_without_git)
+          # Overridden by jtt
+          # %s
+          gem '%s', %p
```

Comment:
it's case by case, sometimes `#{...}` breaks code highlighting in the heredoc string, therefore i prefer format for few arguments. However here it's indeed fine, changing.